.github/workflows/linux.yml

```diff
@@ -13,16 +13,23 @@ jobs:
           - fedora
           - opensuse
           - gentoo
+        cpp_std: [c++17]
+        include:
+          - distro: fedora
+            cpp_std: c++20
+          - distro: alpine
+            ignore-test-errors: true
```

Comment:
Is this because of this https://github.com/Alexays/Waybar/actions/runs/3943499703/jobs/6748390098#step:6:122
?

Reply:
Yes. LC_TIME support on Alpine [is broken](https://wiki.musl-libc.org/open-issues.html#Locale-limitations), but it used to pass because `std::locale` outright reported that `en_US`/`en_GB` were unavailable and the tests were skipped.

Something has changed in Alpine 3.17 (GCC 12?) and now it's possible to get locale facets, except that the localized date formatting is still unsupported. Thus, we have to ignore or skip the tests on Alpine.